common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -718,7 +723,7 @@ def _load_items(self, course_entry, block_keys, depth=0, lazy=True, **kwargs):
         if runtime is None:
             runtime = self.create_runtime(course_entry, lazy)
             self._add_cache(course_entry.structure['_id'], runtime)
-            self.cache_items(runtime, block_keys, course_entry.course_key, depth, lazy)
+            self.cache_items(runtime, block_keys, course_entry.course_key, depth, depth == 0 and lazy)
```

Comment:
Should this check for `depth` be moved within the `cache_items` method?  Would other callers of cache_items also want the same behavior?